web/src/components/task/task-sidebar/task-sidebar.tsx

```diff
@@ -258,37 +257,85 @@ const TaskSidebar: FC<TaskSidebarProps> = ({ jobSettings, viewMode }) => {
         [categories]
     );
 
+    const tabs = ['Categories', 'Data', 'Information', 'Document'];
+
+    const renderTab = (tabName: string) => (
+        <TabButton
+            caption={tabName}
+            isLinkActive={tabValue === tabName}
+            onClick={() => setTabValue(tabName)}
+            size="36"
+        />
+    );
+
+    const taskInfo = (
+        <>
+            <div className={styles['metadata-item']}>
+                <span className={styles['metadata-item__name']}>Document:</span>
+                <span className={styles['metadata-item__value']}>{`${fileMetaInfo.name}`}</span>
+            </div>
+            <div className={styles['metadata-item']}>
+                <span className={styles['metadata-item__name']}>TaskId:</span>
+                <span className={styles['metadata-item__value']}>{`${task?.id}`}</span>
+            </div>
+            <div className={styles['metadata-item']}>
+                <span className={styles['metadata-item__name']}>Pages:</span>
+                <span className={styles['metadata-item__value']}>
+                    {`${task?.pages.join(', ')}`}
+                </span>
+            </div>
+            <div className={styles['metadata-item']}>
+                <span className={styles['metadata-item__name']}>Job Name:</span>
+                <span className={styles['metadata-item__value']}>{`${task?.job.name}`}</span>
+            </div>
+            <div className={styles['metadata-item']}>
+                <span className={styles['metadata-item__name']}>Deadline:</span>
+                <span className={styles['metadata-item__value']}>{`${task?.deadline}`}</span>
+            </div>
+            <div className={styles['metadata-item']}>
+                <span className={styles['metadata-item__name']}>Status:</span>
+                <span className={styles['metadata-item__value']}>{`${task?.status}`}</span>
+            </div>
```

Comment:
It might be better to use a `.map` here to follow the DRY principle. Smth like this:
```   
const arr = [
    { name: "Document:", value: `${fileMetaInfo.name}` },
    { name: "TaskId:", value: `${task?.id}` },
    { name: "Pages:", value: `${task?.pages.join(", ")}` }
  ];
...

arr.map((el) => {
    return (
      <div className={styles["metadata-item"]}>
        <span className={styles["metadata-item__name"]}>{el.name}</span>
        <span className={styles["metadata-item__value"]}>{el.value}</span>
      </div>
    );
  });```